docs/user-guides/metrics.md

```diff
@@ -60,13 +64,22 @@ Amoro has supported built-in metrics to measure status of table self-optimizing
-| optimizer_group_executing_tasks        | Gauge  | group | Number of executing tasks in optimizer group     |
-| optimizer_group_planing_tables         | Gauge  | group | Number of planing tables in optimizer group      |
-| optimizer_group_pending_tables         | Gauge  | group | Number of pending tables in optimizer group      |
-| optimizer_group_executing_tables       | Gauge  | group | Number of executing tables in optimizer group    |
-| optimizer_group_optimizer_instances    | Gauge  | group | Number of optimizer instances in optimizer group |
-| optimizer_group_memory_bytes_allocated | Gauge  | group | Memory bytes allocated in optimizer group        |
-| optimizer_group_threads                | Gauge  | group | Number of total threads in optimizer group       |
+| Metric Name                            | Type  | Tags  | Description                                      |
+|----------------------------------------|-------|-------|--------------------------------------------------|
+| optimizer_group_pending_tasks          | Gauge | group | Number of pending tasks in optimizer group       |
+| optimizer_group_executing_tasks        | Gauge | group | Number of executing tasks in optimizer group     |
+| optimizer_group_planing_tables         | Gauge | group | Number of planing tables in optimizer group      |
+| optimizer_group_pending_tables         | Gauge | group | Number of pending tables in optimizer group      |
+| optimizer_group_executing_tables       | Gauge | group | Number of executing tables in optimizer group    |
+| optimizer_group_optimizer_instances    | Gauge | group | Number of optimizer instances in optimizer group |
+| optimizer_group_memory_bytes_allocated | Gauge | group | Memory bytes allocated in optimizer group        |
+| optimizer_group_threads                | Gauge | group | Number of total threads in optimizer group       |
+
+## Orphan Files Cleaning metrics
+
+| Metric Name                                      | Type    | Tags                     | Description                                                                  |
+|--------------------------------------------------|---------|--------------------------|------------------------------------------------------------------------------|
+| table_orphan_content_cleaning_count              | Counter | catalog, database, table | Count of orphan content files cleaned in the table since ams started         |
+| table_orphan_metadata_file_cleaning_count        | Counter | catalog, database, table | Count of orphan metadata files cleaned in the table since ams started        |
+| table_slated_orphan_content_file_cleaning_count  | Counter | catalog, database, table | Slated Count of orphan content files cleaned in the table since ams started  |
+| table_slated_orphan_metadata_file_cleaning_count | Counter | catalog, database, table | Slated Count of orphan metadata files cleaned in the table since ams started |
```

Comment:
What does `slated_orphan_content_file` and `slated_orpha_metadata_file` means?

Reply:
Indicates the calculated number of files expected to be deleted. Failure may occur during the deletion process, so there will also be an metric of the actual number of deleted files.